scaleway/resource_object_bucket_acl.go

```diff
@@ -162,17 +175,25 @@ func resourceBucketACLCreate(ctx context.Context, d *schema.ResourceData, meta i
 		input.AccessControlPolicy = expandBucketACLAccessControlPolicy(v.([]interface{}))
 	}
 
-	out, err := retryWhenAWSErrCodeEquals(ctx, []string{s3.ErrCodeNoSuchBucket}, &RetryWhenConfig[*s3.PutBucketAclOutput]{
-		Timeout:  d.Timeout(schema.TimeoutCreate),
-		Interval: 5 * time.Second,
-		Function: func() (*s3.PutBucketAclOutput, error) {
-			return conn.PutBucketAclWithContext(ctx, input)
-		},
-	})
-	if err != nil {
-		return diag.FromErr(fmt.Errorf("error putting Object Storage ACL: %s", err))
+	if os.Getenv("TF_ACC") == "1" {
+		out, err := conn.PutBucketAclWithContext(ctx, input)
+		if err != nil {
+			return diag.FromErr(fmt.Errorf("error putting Object Storage ACL: %s", err))
+		}
+		tflog.Debug(ctx, fmt.Sprintf("output: %v", out))
+	} else {
+		out, err := retryWhenAWSErrCodeEquals(ctx, []string{s3.ErrCodeNoSuchBucket}, &RetryWhenConfig[*s3.PutBucketAclOutput]{
+			Timeout:  d.Timeout(schema.TimeoutCreate),
```

Comment:
If the timeout is too long we should have a way to bypass it in the tests https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/retries-and-customizable-timeouts